src/migrations/043_userinfo_oidc.ts

```diff
@@ -0,0 +1,56 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  const result = await knex('changelog').select('id').where({ id: 43 });
+
+  if (result.length) {
+    // Migration has been applied using the old patch system
+    return;
+  }
+
+  await knex('changelog').insert({
+    id: 43,
+    timestamp: Math.floor(Date.now() / 1000)
+  });
+
+  await knex.schema.createTable('user_info', table => {
+    table.increments();
+    table.string('sub', 50).notNullable();
+    table.string('name', 200).nullable();
+    table.string('email', 100).nullable();
+    table.boolean('email_verified').nullable();
```

Comment:
I kind of went into this in #624, but only a few fields are actually needed. A lot of these fields already in the database in some form, so you can remove everything that wasn't already listed in #624, and it would be nice to add `metadata` for custom fields

Reply:
> A lot of these fields already in the database in some form

Should I reference them with... refs, queries or foreign keys?

Reply:
No I meant check out the ticket and remove all fields that aren't mentioned there. We don't want data to be in our database duplicated

Reply:
Ohh ok! a schema is not like a query view! lol

Reply:
No it's a database table